.github/workflows/build_and_publish_docker_image.yml

```diff
@@ -0,0 +1,59 @@
+name: build_and_publish_docker_image
+
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch:
+
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  build_and_publish_image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          registry: docker.io
+          username: ${{ vars.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      
+      - name: Extract Docker metadata
+        uses: docker/metadata-action@v5
+        id: meta
+        with:
+          images: ${{ github.repository }}
+          flavor: |
+            latest=auto
+          tags: |
+            type=raw,value=latest,enable={{is_default_branch}}
+            type=semver,pattern={{version}}
```

Comment:
This is different from `update_dashboard.yml`:
```
        - name: Extract Docker metadata
          uses: docker/metadata-action@v5
          with:
            images: ghcr.io/${{ github.repository }}/superset_df_dashboard
            flavor: |
              latest=auto
            tags: |
              type=raw,value=latest,enable={{is_default_branch}}
```
Could you confirm it's working?